parachains/runtimes/assets/statemine/tests/tests.rs

```diff
@@ -302,3 +322,70 @@ fn test_that_buying_ed_refund_does_not_refund() {
 			assert_eq!(Assets::total_supply(1), ExistentialDeposit::get());
 		});
 }
+
+#[test]
+fn test_asset_xcm_trader_not_possible_for_non_sufficient_assets() {
+	ExtBuilder::<Runtime>::default()
+		.with_collators(vec![AccountId::from(ALICE)])
+		.with_session_keys(vec![(
+			AccountId::from(ALICE),
+			AccountId::from(ALICE),
+			SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) },
+		)])
+		.build()
+		.execute_with(|| {
+			// Create a non-sufficient asset with specific existential deposit
+			let minimum_asset_balance = 1_000_000_u128;
+			assert_ok!(Assets::force_create(
+				RuntimeHelper::<Runtime>::root_origin(),
+				1,
```

Comment:
Let's also extract this into a `local_asset_id` binding.

Reply:
@KiChjang 
there is a following issue: https://github.com/paritytech/cumulus/issues/1915,
so I will fix your comment there plus check all other tests